pages/tutorials/[slug].js

```diff
@@ -30,10 +34,50 @@ export async function getStaticProps(context) {
 }
 
 export default function Tutorial({ name, steps, id, slug }) {
+  const [currentStep, setCurrentStep] = useState(0);
+
+  function addStep() {
+    setCurrentStep((prevCurrentStep) =>
+      prevCurrentStep < steps.length ? prevCurrentStep + 1 : prevCurrentStep
+    );
+  }
+
+  function subtractStep() {
+    setCurrentStep((prevCurrentStep) =>
+      prevCurrentStep > 0 ? prevCurrentStep - 1 : 0
+    );
+  }
+
   return (
     <>
       <Headline>{name}</Headline>
-      <TutorialStartCard />
+
+      {currentStep === 0 ? (
+        <TutorialStartCard />
+      ) : currentStep < steps.length ? (
+        <TutorialCard step={steps[currentStep]} />
+      ) : (
+        <Headline>done</Headline>
+      )}
+
+      <ButtonContainer>
+        {currentStep === 0 ? (
+          <>
+            <Link href="/tutorials">
+              <Button isPrimary={false}>back to library</Button>
+            </Link>
+          </>
+        ) : (
+          <Button isPrimary={false} onClick={() => subtractStep()}>
+            prev
+          </Button>
+        )}
+        {currentStep !== steps.length && (
+          <Button isPrimary={true} onClick={() => addStep()}>
+            {currentStep === 0 ? "start" : "next"}
```

Comment:
very optional 2: i think id like as a user to just have at the last step a 'next' button called 'done' (which brings me back to home or the tutorial menu) instead of clicking on 'next' and being relocated to another page and then clicking on another button.

Reply:
I see, you're eager to learn and directly want to proceed to your next learning contents – that's fantastic! 🎉 
I ask you for a little more patience for this feature but here is a little spoiler: It is already in the planning for my next user story!